pom.xml

```diff
@@ -2000,234 +2000,12 @@
              To run tests against a different version of Elasticsearch, use the appropriate profile
              and provide the Elasticsearch version:
 
-                 mvn clean test -Pelasticsearch-2.0 -Dtest.elasticsearch.version=2.1.0
+                 mvn clean test -Pelasticsearch-6.0 -Dtest.elasticsearch.version=6.0.0
           -->
 
-        <!-- Elasticsearch [2.0,2.2) test environment -->
+        <!-- Elasticsearch 5.6+ test environment (default) -->
```

Comment:
Shouldn't the latest stable be the default?

Reply:
Ultimately it should, but right now this would break the integration tests in the legacy code, so... Let's do that when we get rid of the legacy code?

Reply:
Note we execute tests against 6.0 regardless, that's just a matter of cleanliness.